app/main.py

```diff
@@ -1,6 +1,17 @@
 from typing import Callable
 
 
-def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+def cache(*func: Callable) -> Callable:
```

Comment:
The `cache` function is defined to accept a variable number of `Callable` arguments, but the unpacking operator `*` is incorrectly used in the parameter list. It should be `func: Callable` without the `*`.